h2o-py/h2o/model/extensions/fairness.py

```diff
@@ -420,6 +432,9 @@ def inspect_model_fairness(self, frame, protected_columns, reference, favorable_
         >>> gbm.inspect_model_fairness(test, protected_columns, reference, favorable_class)
         """
         import h2o
+        from h2o.utils.shared_utils import can_use_numpy
+        if not can_use_numpy():
+            raise ImportError("numpy is required for inspect_model_fairness.")
```

Comment:
Doesn't use `numpy` directly but indirectly so it's not imported in this function but it is required.